VetClinic.WebApi.Tests/Controllers/PetsControllerTests.cs

```diff
@@ -149,6 +152,7 @@ public void CanGetAllPets()
         [Fact]
         public void CanGetPetById()
         {
+            // Arrange
             var pets = GetTestPets().AsQueryable();
             var id = 10;
             var name = "Lord10";
```

Comment:
why would you need this variable?

Reply:
what variable?  

this?=>var name = "Lord10";

Assert.Equal(name, model.Name);

Reply:
yes